ios/FluentUI/Vnext/Avatar/Avatar.swift

```diff
@@ -224,6 +226,24 @@ public struct AvatarView: View {
             }
         }
 
+        @ViewBuilder
+        var avatarRingView: some View {
```

Comment:
i understand in avatarBody you check if the style is group but should there be a check or comment this variable is not available in group style?

Reply:
The job of the check is being done by the one you mentioned in the avatarbody computed variable. The avatarRingView is not computed unless it's being requested in the avatarBody.
I can definitely add a comment to make it explicit that the ring is not supported in the group style.